tox.ini

```diff
@@ -71,6 +71,7 @@ passenv =
     dev: JSTOR_API_URL
     dev: JSTOR_API_SECRET
     dev: DISABLE_KEY_ROTATION
+    dev: ADMIN_USERS
```

Comment:
Hmm, yeah I guess there isn't really a good default admin user we could add in dev, since each developer uses their own Google account, so if you want to test superadmin mode in dev you're gonna have to set `ADMIN_USERS` to your own account?

Reply:
Heh, https://github.com/hypothesis/devdata/pull/100